common/djangoapps/static_replace/__init__.py

```diff
@@ -109,6 +110,13 @@ def wrap_part_extraction(match):
         prefix = match.group('prefix')
         quote = match.group('quote')
         rest = match.group('rest')
+
+        # Don't rewrite XBlock resource links.  Probably wasn't a good idea that /static
+        # works for actual static assets and for magical course asset URLs....
+        full_url = prefix + rest
```

Comment:
We concatenate these two pieces because `original` actually includes the double quotes, and rather than manipulating that.. I know that `prefix` will be `/static` and `rest` will be `/xblock/....`, which gives me the full URL without the quotes, no potentially dubious manipulation required.